deploy.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+if ([ $TRAVIS_PULL_REQUEST == "false" ])
```

Comment:
Yes, I didn't think about it, but I guess we don't want this to break travis on peoples forks. Will add.